projects/cloud/app/frontend/stores/OrganizationStore.ts

```diff
@@ -0,0 +1,111 @@
+import { ApolloClient } from '@apollo/client';
+import { makeAutoObservable, runInAction } from 'mobx';
+import {
+  ChangeUserRoleDocument,
+  OrganizationQuery,
+  OrganizationDocument,
+  Role,
+} from '../graphql/types';
+
+class OrganizationStore {
+  organization: OrganizationQuery['organization'];
+
+  client: ApolloClient<object>;
+
+  constructor(client: ApolloClient<object>) {
+    this.client = client;
+    makeAutoObservable(this);
```

Comment:
Does the position of `makeAutoObservable` in the constructor matter? I've seen `HomeStore` has this one as the first line.

Reply:
the docs show it on the [first](https://mobx.js.org/observable-state.html#makeautoobservable) line, so I'd default to that convention - although I am not sure if the result will be different. Will change it, though 👍